src/react-apps/applications/service-development/src/store/index.ts

```diff
@@ -18,6 +18,8 @@ function configureStore(initialState?: any): Store<IServiceDevelopmentState> {
   let enhancer: any;
 
   if (process.env.NODE_ENV === 'development') {
+    const { logger } = require('redux-logger');
+    middlewares.push(logger);
```

Comment:
Are we going to have this back in service-development?

Reply:
I commented out the code so we can use it when we need.